chainer/links/model/vision/vgg.py

```diff
@@ -66,26 +68,33 @@ class VGG16Layers(link.Chain):
             are not initialized by the pre-trained model, but the default
             initializer used in the original paper, i.e.,
             ``chainer.initializers.Normal(scale=0.01)``.
+        n_layers (int): The number of layers of this model. It should be
+            either 16 or 19.
 
     Attributes:
         available_layers (list of str): The list of available layer names
             used by ``forward`` and ``extract`` methods.
 
     """
 
-    def __init__(self, pretrained_model='auto'):
+    def __init__(self, pretrained_model='auto', n_layers=16):
+        super(VGGLayers, self).__init__()
```

Comment:
Why is this code moved?

Reply:
Because I added the comment "You can use ... VGG16Layers and VGG19Layers" at the beginning of the above docstring.

Reply:
If i mistakenly put the explanation in wrong place, please tell me the exact/appropriate line. +)

Reply:
Ok, I initially meant to ask why the line `super(VGGLayers, self).__init__()` has been moved, but the current code is also fine.